webapp/src/views/projects/translations/KeyCreateForm/FormBody.tsx

```diff
@@ -0,0 +1,268 @@
+    projectId: project.id,
+    baseText,
+    targetLanguageId: targetLang?.id as number,
+    keyId: undefined as any,
+    onValueUpdate,
+    enabled: hintRelevant,
+  });
+
+  const classes = useStyles();
+  return (
+    <>
+      <div className={classes.container}>
+        <FastField name="name">
+          {({ field, form, meta }: FieldProps<any>) => {
+            return (
+              <div>
+                <FieldLabel>
+                  <T>translation_single_label_key</T>
+                </FieldLabel>
+                <div className={classes.field}>
+                  <div
+                    className={classes.editorWrapper}
+                    data-cy="translation-create-key-input"
+                  >
+                    <Editor
+                      plaintext
+                      value={field.value}
+                      onChange={(val) => {
+                        form.setFieldValue(field.name, val);
+                      }}
+                      onSave={() => form.handleSubmit()}
+                      onBlur={() => form.setFieldTouched(field.name, true)}
+                      minHeight="unset"
+                      autofocus={autofocus}
+                      scrollMargins={{ bottom: 150 }}
+                      autoScrollIntoView
+                    />
+                  </div>
+                </div>
+                <Typography
+                  color="error"
+                  variant="caption"
+                  className={classes.error}
+                >
+                  {meta.touched && meta.error}
+                </Typography>
+              </div>
+            );
+          }}
+        </FastField>
+
+        <FieldArray
+          name="tags"
+          render={(helpers) => (
+            <FastField name="tags">
+              {({ field }: FieldProps<any>) => {
+                return (
+                  <div>
+                    <FieldLabel>
+                      <T>translation_single_label_tags</T>
+                    </FieldLabel>
+                    <div className={classes.tags}>
+                      {field.value.map((tag, index) => {
+                        return (
+                          <Tag
+                            key={tag}
+                            name={tag}
+                            onDelete={() => helpers.remove(index)}
+                          />
+                        );
+                      })}
+                      <TagInput
+                        existing={field.value}
+                        onAdd={(name) =>
+                          !field.value.includes(name) && helpers.push(name)
+                        }
+                        placeholder={t('translation_single_tag_placeholder')}
+                      />
+                    </div>
+                  </div>
+                );
+              }}
+            </FastField>
+          )}
+        />
+        {languages.map((lang) => {
+          return (
```

Comment:
The curly brackets and return is useless. Also it would be lice to move the body to separate component.